generator/README.md

```diff
@@ -34,6 +34,8 @@ Generates data to a specified endpoint.
 * `<profile path>`, a path to the profile JSON file
 * `<output path>`, a file path to where the data should be emitted to. This will be a UTF-8 encoded CSV file or directory, option dependent.
 
+As standard the velocity monitor is implemented.
```

Comment:
I don't think there is a need for this statement, its clarified in the generator options document.